src/components/POIForm.component.js

```diff
@@ -71,29 +71,26 @@ class POIForm extends Component {
   onSubmit() {}
 
   render() {
+    const stories = this.props.stories
+
     return (
       <Form horizontal>
         <PageHeader>
           POI Input Form
-          <small>Complete each component below</small>
+          <small> Complete each component below</small>
         </PageHeader>
 
         {/*
           *********THIS IS MY POI NAME***********
```

Comment:
I think you can remove these comments now - there's enough context given in the `FieldGroup` props that now the comments don't tell us anything we don't already know